Tests/GRDBTests/JSONColumnTests.swift

```diff
@@ -1,5 +1,8 @@
 import XCTest
 import GRDB
+#if GRDBCIPHER
+import SQLCipher
+#endif
```

Comment:
To be removed if we replace `sqlite3_libversion_number()` with `Database.sqliteLibVersionNumber`.